src/main/java/graphql/schema/FieldCoordinates.java

```diff
@@ -62,7 +84,9 @@ public String toString() {
      * @return new field coordinates represented by the two parameters
      */
     public static FieldCoordinates coordinates(GraphQLFieldsContainer parentType, GraphQLFieldDefinition fieldDefinition) {
-        return new FieldCoordinates(parentType.getName(), fieldDefinition.getName());
+        final String parentTypeName = parentType.getName();
+        return new FieldCoordinates(parentTypeName, fieldDefinition.getName(),
+                (null == parentTypeName) || parentTypeName.isEmpty());
```

Comment:
I'm not sure whether this is correct or not.  I'm making the assumption that if the parent type has no name or the name is empty that we should consider this a system coordinate, similar to what is done in the string method

Reply:
Just put `false`

The only true way to create a system coord is this method

```
    public static FieldCoordinates systemCoordinates(String fieldName)
```

Reply:
Please fix this one and this fix is ready to go

Reply:
Thanks, will do.